src/DeviceInterface/ExtensionInterface.js

```diff
@@ -202,12 +222,17 @@ class ExtensionInterface extends InterfacePrototype {
     }
 
     addLogoutListener (handler) {
+        if (this._logoutListenerHandler) {
+            chrome.runtime.onMessage.removeListener(this._logoutListenerHandler)
+        }
```

Comment:
Can you add a comment explaining why we need to remove a previous listener if set? I think I get it now, but I'll surely forget in 10 minutes 🙂.

Reply:
@GioSensation We only ever want one log out listener and this method is called in `postInit`, which means resetting autofill will cause another listener to be attached. In that case, we want to remove the previous logout listener first.

Reply:
Yeah, if you can just add it as a comment in the code it'll be easier for our future selves 🙂.